src/sentry/data_export/endpoints/data_export.py

```diff
@@ -61,13 +61,17 @@ def post(self, request, organization):
             )
             status = 200
             if created:
-                metrics.incr("dataexport.start", tags={"query_type": data["query_type"]})
+                metrics.incr(
+                    "dataexport.enqueue", tags={"query_type": data["query_type"]}, sample_rate=1.0
```

Comment:
changed `dataexport.start` to `dataexport.enqueue` since it overlaps with the actual `start` metric of when we begin assembling the CSV